components/sidebar/sidebar.jsx

```diff
@@ -234,7 +234,7 @@ export default class Sidebar extends React.PureComponent {
 
     setBadgesActiveAndFavicon() {
         this.lastBadgesActive = this.badgesActive;
-        this.badgesActive = this.props.unreads.mentions;
+        this.badgesActive = this.props.unreads.mentionCount;
```

Comment:
Not related but fix since the original props.unreads doesn't have `mentions` property.  It's actually `mentionCount`.